content/blog/announcing-http-add-on.md

```diff
@@ -0,0 +1,142 @@
+- **External scaler**: This is an [external push scaler](https://keda.sh/docs/2.1/scalers/external-push/) that constantly pings the interceptor to get pending HTTP queue metrics. It transforms these data and sends them directly down to KEDA, which then makes a scaling decision
+- ***Operator**:  The operator is a component that exists just for the convenience of the user. It listens for new [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) resources, HTTPScaledObject, and wire up your workloads to support autoscaling. The operator will create & configure the interceptor and scaler for you such that a target Deployment that you specify will begin autoscaling based on incoming HTTP traffic.
+
+![architecture diagram](/static/img/blog/http-add-on/arch.png)
+
+## Seeing it in action
+
+We’ve aimed to make the HTTP add-on approximately as simple to operate and use as KEDA. 
+
+There are two major steps to using it:
+
+1. Install the KEDA HTTP add-on operator
+2. Create a new `HTTPScaledObject`
+
+### Installing the Operator
+
+This is an optional step, but we highly recommend it because the operator makes interacting with the HTTP add-on more convenient. We’ve packaged the operator up in a [Helm](https://helm.sh) chart, which you can install by following the instructions below.
+
+_The commands below have been tested on Mac and Linux (Ubuntu). They likely work on Windows WSL2, but won’t work on Powershell. You’ll also need to ensure you have access to a Kubernetes cluster with a Kubernetes configuration file properly configured._
+
+#### Install KEDA
+
+First, make sure you’ve set up an environment variable for the namespace you’d like to install everything into:
+
+```shell
+export NAMESPACE=mynamespace
+```
+
+Next, install KEDA:
+
+```shell
+helm repo add kedacore https://kedacore.github.io/charts
+helm repo update
+helm install --create-namespace -n ${NAMESPACE} keda kedacore/keda
+```
+
+#### Install the Operator
+
+Now that KEDA is installed, you can install the operator. We've packaged it up in [Artifact Hub](https://artifacthub.io/packages/keda-scaler/keda-official-external-scalers/keda-add-ons-http) to make it easy to install without checking out any code.
+
+TODO: instructions here for using artifact hub
+
+### Install an `HTTPScaledObject`
+
+The `HTTPScaledObject` CRD instructs the operator to install and configure the interceptor and scaler for a specified `Deployment`. For this to work, you’ll need to have a Deployment running in the same `$NAMESPACE` and a Service configured to route traffic to the pods in that `Deployment`.
+
+>We've created a sample application [here](TODO) as a Helm chart for you, if you'd like to install it. It comes complete with a properly configured `HTTPScaledObject`, so when you install the chart, everything will be automatically configured for you and you can skip the rest of this section.
```

Comment:
TODO: add link to sample application after https://github.com/kedacore/http-add-on/pull/125 is merged (the app will have moved to the `examples/` directory

Reply:
Done